librosa/display.py

```diff
@@ -617,6 +618,19 @@ def __envelope(x, hop):
     return x_frame.max(axis=1)
 
 
+_categorical_axis_types = ['chroma', 'chroma_h', 'tonnetz', 'frames']
```

Comment:
Not all categorical axes should be considered "equal".  Probably only chroma and chroma_h make sense here.

frames is a time-like axis, but really is only compatible with itself.